include/termcolor/termcolor.hpp

```diff
@@ -202,6 +202,41 @@ namespace termcolor
         return stream;
     }
 
+    template <int code>
+    inline
+    std::ostream& fg_color(std::ostream& stream)
```

Comment:
In order to be consistent with existing naming convention, can you please rename `fg_color` into `color` and `bg_color` into `on_color`?